plotter/polygon.go

```diff
@@ -0,0 +1,118 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"image/color"
+	"math"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+)
+
+// Polygon implements the Plotter interface, drawing a polygon.
+type Polygon struct {
+	// XYs is a copy of the vertices of this polygon.
+	XYs []XYs
+
+	// LineStyle is the style of the line around the edge
+	// of the polygon.
+	draw.LineStyle
+
+	// Color is the fill color of the polygon.
+	Color color.Color
```

Comment:
This is potentially different for different backends. One of the tests documents the behavior for the img backend. I can add some text to clarify this and also tests for the other default backends.